install/changelog.md

```diff
@@ -52,6 +57,86 @@ title: "Change Log"
 
 > See [known issues](../reference/known-issues.html) for currently known issues and workarounds.
 
+## v0.15.0 {#v150}
+
+Released on August 13, 2018
+
+In addition to the 0.15.0 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
+
+This release brings many large improvements to the Pulumi CLI. Major features include:
+
+#### Parallelism
+
+Pulumi now preforms resource operations in parallel, when possible. By default, Pulumi allows ten parallel operations, but this can be controlled with the `-p` flag to `pulumi`. Presently, deletes are still processed sequentaly, but this will be changed in a future release. It is possible that your program may have implicit dependencies between two resources and the change to preform operations in parallel by default will cause issues. If this happens, we recommend that you set dependsOn on property in `resourceOptions` parameter to any resource. You may also pass `-p=1` to disable parallelism. Parallelism is only supported for nodejs programs today.
```

Comment:
Does `-p=1` disable parallelism, or is that `=0`?  @swgillespie?

Reply:
Preforms => performs.

Reply:
Anything <= 1 disables parallelism.